tsdb/chunks/head_chunks.go

```diff
@@ -860,8 +887,13 @@ func (cdm *ChunkDiskMapper) IterateAllChunks(f func(seriesRef HeadSeriesRef, chu
 			if maxt > mmapFile.maxt {
 				mmapFile.maxt = maxt
 			}
+			isOOO := cdm.IsOutOfOrderChunk(chkEnc)
+			// flip the first bit to restore the original encoding
+			if isOOO {
+				chkEnc = cdm.UnapplyOutOfOrderMask(chkEnc)
+			}
```

Comment:
Same reword as the other comment + don't need the `if` block.
```suggestion
			// Extract the encoding from the byte. ChunkDiskMapper uses only the last 7 bits for the encoding.
			chkEnc = cdm.UnapplyOutOfOrderMask(chkEnc)
```

Reply:
Done